From: Iain Lane <laney@debian.org>
Date: Tue, 3 Sep 2019 16:25:39 +0100
Subject: Revert "build: Include libdir in rpath"

This reverts commit a92309f9cf9099a8b0b54908e032dcb181ab6c63.

We don't want these in rpath on Debian; the standard libdir is in the
search path.

Forwarded: not-needed
---
 meson.build             | 1 -
 src/portal/meson.build  | 2 +-
 src/tracker/meson.build | 2 +-
 3 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index 2c1fc9e..ce7959f 100644
--- a/meson.build
+++ b/meson.build
@@ -292,7 +292,6 @@ build_root = meson.current_build_dir()
 po_subdir = join_paths(source_root, 'po')
 
 tracker_internal_libs_dir = join_paths(get_option('prefix'), get_option('libdir'), 'tracker-' + tracker_api_version)
-tracker_install_rpath = ':'.join([tracker_internal_libs_dir, libdir])
 
 # NOTE: We don't use ${TRACKER_API_VERSION} because other content like
 # the ontology is installed to the same location.
diff --git a/src/portal/meson.build b/src/portal/meson.build
index b4bb7fe..6c0b201 100644
--- a/src/portal/meson.build
+++ b/src/portal/meson.build
@@ -8,7 +8,7 @@ sources = [
 executable('tracker-xdg-portal-@0@'.format(tracker_api_major), sources,
     c_args: tracker_c_args,
     install: true,
-    install_rpath: tracker_install_rpath,
+    install_rpath: tracker_internal_libs_dir,
     install_dir: get_option('libexecdir'),
     dependencies: [tracker_sparql_dep, tracker_common_dep],
     include_directories: [commoninc, configinc, srcinc],
diff --git a/src/tracker/meson.build b/src/tracker/meson.build
index 882945a..1cb0a49 100644
--- a/src/tracker/meson.build
+++ b/src/tracker/meson.build
@@ -26,7 +26,7 @@ executable(command_name, sources,
         '-DBINDIR="@0@"'.format(join_paths(get_option('prefix'), get_option('bindir'))),
     ],
     install: true,
-    install_rpath: tracker_install_rpath,
+    install_rpath: tracker_internal_libs_dir,
     dependencies: [tracker_sparql_dep, tracker_data_dep],
     include_directories: [commoninc, configinc, srcinc],
 )
